### PR TITLE
Fixes #12393: Add docs about Windows agent logging

### PR DESCRIPTION
**Pull request by Fdall**
https://www.rudder-project.org/redmine/issues/12393

### CODE REVIEW

File: 95_windows/70_agent_cli.txt [hunk 1]
```diff
@@ -29,3 +29,39 @@ where action can be one of the following
...
+
+Rudder logs are visible in the output of the agent. You can get more details about what is done with the 
+`-Verbose` option:
+
+----
+rudder agent run -v
+----
+
+You can also explore all agent logs (including those from unattended runs) in the Windows Event Viewer.
+Before plugin version 4.2-1.6 Rudder used the windows system eventlog and was logging in
```
Comment (amousset) [inline 95_windows/70_agent_cli.txt]: Windows

File: 95_windows/70_agent_cli.txt [hunk 2]
```diff
@@ -29,3 +29,39 @@ where action can be one of the following
...
+rudder agent run -v
+----
+
+You can also explore all agent logs (including those from unattended runs) in the Windows Event Viewer.
+Before plugin version 4.2-1.6 Rudder used the windows system eventlog and was logging in 
+the *Windows Logs -> Application* view, with the *Rudder* source and the *101* Event ID.
+
+Since the Windows plugin version 4.2-1.6 Rudder will report in a dedicated windows journal named Rudder and its logs are saved on different verbosity:
+
+* *classic rudder reports* will have the *Event ID 101*, they are the reports sent to the server.
```
Comment (amousset) [inline 95_windows/70_agent_cli.txt]: Rudder reports

File: 95_windows/70_agent_cli.txt [hunk 3]
```diff
@@ -29,3 +29,39 @@ where action can be one of the following
...
+
+* *Info level reports* will have the *Event ID 102* and will only be local logs.
+
+If you had an old plugin version installed rudder will not try to install the new journal reference because
```
Comment (amousset) [inline 95_windows/70_agent_cli.txt]: Rudder

File: 95_windows/70_agent_cli.txt [hunk 4]
```diff
@@ -29,3 +29,39 @@ where action can be one of the following
...
+it needs a complete reboot of the host system. 
+See the last note on the Microsoft doc: https://msdn.microsoft.com/en-us/library/2awhba7a%28v=vs.110%29.aspx.
+
+If you want to change manually the Rudder eventlog use the following process, keep in mind that it will need a machine restart to avoid any rpeorting issues.
```
Comment (amousset) [inline 95_windows/70_agent_cli.txt]: reporting issues

File: 95_windows/70_agent_cli.txt [hunk 5]
```diff
@@ -29,3 +29,39 @@ where action can be one of the following
...
+First identify the current eventlog for Rudder by running in the powershell console
+----
+[System.Diagnostics.EventLog]::LogNameFromSourceName("Rudder", ".")
+----
+If it does not suit you, remove the rudder source from it and create a new logger for Rudder
```
Comment (amousset) [inline 95_windows/70_agent_cli.txt]: Rudder source

**Comment by Fdall** [comment]
Commit modified

**Pull request merged**